packages/docs/src/components/MedplumCodeBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import CodeBlock, { Props } from '@theme/CodeBlock';
+import * as React from 'react';
+
+const BLOCK_START_PATTERN = new RegExp('^//\\s*start-block\\s+([A-Za-z_-]*)', '');
+// const BLOCK_START_PATTERN = /^\/\/\s*start-block\s+([A-Za-z_-]*)/;
+const BLOCK_END_PATTERN = new RegExp('^//\\s*end-block\\s+([A-Za-z_-]*)', '');
+
+interface MedplumCodeBlockProps extends Props {
+  selectLines?: number[][];
+  selectBlocks?: string[] | string;
```

Comment:
Keep it simple and remove the `string[]`?  All examples in `create-fhir-data.md` use single string, yeah?